monday.tex

```diff
@@ -433,8 +433,8 @@ \section*{Monday Sessions Descriptions}\label{monday-descriptions}
   Iranian OSM Community, last two years and the way up front~\noVideo%
 }{%
 }{%
-  We are going to review the last two years of Iranian OSM community challenges, progresses, growth
-  and the fast expansion in Organised Editing, also, we will discuss about our future plans.%
+  We review the last 2 years of Iranian OSM Community challenges, progresses, growth,
```

Comment:
I thought it is good style two write out small numbers (up to ten).

Reply:
Do you understand the "Community" in "Iranian OSM Community" as a or a part of a proper noun which requires capitalisation?